src/components/ConnectionsTable/ConnectionsList.module.scss

```diff
@@ -22,6 +22,7 @@
   padding-top: 0.2rem;
   padding-bottom: 0.2rem;
   @include styles.code-text;
+  font-size: smaller;
   font-style: italic;
```

Comment:
This text style is probably shared with another part of the GUI so we should make it global in the future, pls add a `//TODO: make global`